src/components/PrimaryPanes/SourcesTreeItem.js

```diff
@@ -249,6 +249,7 @@ class SourceTreeItem extends Component<Props, State> {
         key={item.path}
         onClick={this.onClick}
         onContextMenu={e => this.onContextMenu(e, item)}
+        role="presentation"
```

Comment:
This does not seem to be for presentation only since there's a click and context menu handlers. It should probably have some semantic meaning.